internal/command/migrate_to_v2/migrate_to_v2.go

```diff
@@ -206,11 +214,22 @@ func NewV2PlatformMigrator(ctx context.Context, appName string) (V2PlatformMigra
 		recovery: recoveryState{
 			platformVersion: appFull.PlatformVersion,
 		},
+		primaryRegion: appConfig.PrimaryRegion,
 	}
-	err = migrator.validate(ctx)
-	if err != nil {
-		return nil, err
+	if region, ok := migrator.appConfig.Env["PRIMARY_REGION"]; ok {
+		migrator.primaryRegion = region
 	}
+	if migrator.isPostgres {
+		consul, err := apiClient.EnablePostgresConsul(ctx, appCompact.Name)
+		if err != nil {
+			return nil, err
+		}
+		migrator.pgConsulUrl = consul.ConsulURL
+	}
+	//err = migrator.validate(ctx)
+	//if err != nil {
+	//	return nil, err
+	//}
```

Comment:
assuming this and the other commented out block in here are in in-development-thing-i-don't-need-to-worry-about land and won't end up affecting app migrations